src/client/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { format } from 'date-fns'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { Telephone } from '../icons/Telephone'
+
+type Props = {
+  color?: 'black' | 'white'
+}
+
+const Wrapper = styled.div<Props>`
+  color: ${(props) =>
+    props.color === 'black' ? colorsV3.black : colorsV3.white};
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  && {
+    margin-right: 0.4rem;
+  }
+`
+
+const PhoneLink = styled.a`
+  text-decoration: none;
+  color: inherit;
+  font-size: 0.875rem;
+
+  :hover {
+    color: ${colorsV3.gray600};
+  }
+`
+
+const Text = styled.p<Props>`
+  color: ${(props) =>
```

Comment:
Same here - destructuing props makes the code easier to read, at least for me. But it anyone wants to challenge that, of course go ahead 😇